tests/conftest.py

```diff
@@ -18,7 +18,6 @@
 EASYPOST_PROD_API_KEY = os.getenv("EASYPOST_PROD_API_KEY")
 PARTNER_USER_PROD_API_KEY = os.getenv("PARTNER_USER_PROD_API_KEY", "123")
 REFERRAL_CUSTOMER_PROD_API_KEY = os.getenv("REFERRAL_CUSTOMER_PROD_API_KEY", "123")
-STRIPE_CONNECT_USER_PROD_API_KEY = os.getenv("STRIPE_CONNECT_USER_PROD_API_KEY", "123")
```

Comment:
Question: why was this removed?

Reply:
It was added previously when I was fixing the partner tests but wasn't needed after all. The fixture was removed earlier but the constant remained and is unused so I removed it.